packages/core/src/RenderingEngine/VolumeViewport.ts

```diff
@@ -25,6 +25,7 @@ import BaseVolumeViewport from './BaseVolumeViewport';
 import setDefaultVolumeVOI from './helpers/setDefaultVolumeVOI';
 import { setTransferFunctionNodes } from '../utilities/transferFunctionUtils';
 import { ImageActor } from '../types/IActor';
+import { utilities } from '..';
```

Comment:
this will usually cause a circular dependency, better to import the actual utility you need yourself